articles/provider-quantinuum.md

```diff
@@ -63,9 +67,9 @@ After validating the syntax of their code with a Syntax Checker, users can take
 More information can be found in the *System Model H1 Emulator Product Data Sheet* found on the [System Model H1] page.
 
 - Job type: `Simulation`
-- Data Format: `honeywell.openqasm.v1`, `honeywell.qir.v1`
-- Target ID: 
-  - H1-1 Emulator: `quantinuum.sim.h1-1e` 
+- Data Format: `quantinuum.openqasm.v1`
```

Comment:
Should the data format for the syntax checkers and the Model H1 also change to quantinuum.openqasm.v1?